libraries/chain/include/eosio/chain/asset.hpp

```diff
@@ -14,63 +15,64 @@
 
 namespace eosio { namespace chain {
 
-using asset_symbol = uint64_t;
+      //using asset_symbol = uint64_t;
 
 struct asset
 {
-   asset(share_type a = 0, asset_symbol id = EOS_SYMBOL)
-      :amount(a),symbol(id){}
+   asset(share_type a = 0, symbol id = EOS_SYMBOL)
+      :amount(a), sym(id){}
```

Comment:
Add `explicit` to constructor.

Reply:
Done. This broke the code in quite a few places. I fixed those.